scripts/linux/openc3_test.sh

```diff
@@ -27,6 +27,46 @@ case $1 in
     rspec
     cd -
     ;;
+
+  playwright )
+    case $2 in
+      install-playwright )
+        cd playwright
+        ./playwright.sh install-playwright
```

Comment:
Not sure if it matters, but other places where we cd into a folder we cd - out at the end.

Reply:
I went ahead and added `cd -` in the places where I `cd` into a directory.